ci/build-ignore-timeouts.sh

```diff
@@ -1,29 +0,0 @@
-#!/bin/bash -x
-
-output=$(mktemp)
-
-mkdir -p book/
-cp -r $HOME/linkcheck/ book/
-RUST_LOG=mdbook_linkcheck=debug mdbook-linkcheck -s 2>&1 | tee -a $output
-cp -r book/linkcheck $HOME/
-
-mdbook build
-
-result=${PIPESTATUS[0]}
```

Comment:
We need to keep at least this part because this is what invokes the linkchecker.